src/Common/src/System/Security/Cryptography/Asn1/PrivateKeyInfoAsn.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<asn:Sequence
+  xmlns:asn="http://schemas.dot.net/asnxml/201808/"
+  name="PrivateKeyInfoAsn"
+  namespace="System.Security.Cryptography.Asn1">
+
+  <!--
+    https://tools.ietf.org/html/rfc5208#section-5
+
+    PrivateKeyInfo ::= SEQUENCE {
+        version                   Version,
+        privateKeyAlgorithm       PrivateKeyAlgorithmIdentifier,
+        privateKey                PrivateKey,
+        attributes           [0]  IMPLICIT Attributes OPTIONAL
+    }
+
+    Version ::= INTEGER
+    PrivateKeyAlgorithmIdentifier ::= AlgorithmIdentifier
+    PrivateKey ::= OCTET STRING
+    Attributes ::= SET OF Attribute
+  -->
+  <asn:Integer name="Version" backingType="byte" />
+  <asn:AsnType name="PrivateKeyAlgorithm" typeName="System.Security.Cryptography.Asn1.AlgorithmIdentifierAsn" />
+  <asn:OctetString name="PrivateKey" />
+  <asn:SetOf name="Attributes" implicitTag="0" optional="true">
+    <asn:AsnType typeName="System.Security.Cryptography.Asn1.AttributeAsn" />
+  </asn:SetOf>
```

Comment:
This was incorrectly declare as `SequenceOf` before the PR. There are no unit tests for it, unfortunately.

Reply:
Pkcs8PrivateKeyInfoTests.EnsureAttributesParse sort of does.  But since it has an implicit tag all that reading it as a `SEQUENCE OF` instead of a `SET OF` did is bypass the sorted data check (if reading in DER mode).  Writing it might have still treated it as a `SET OF`, because it might have run through the NormalizeAttributes code.

I don't know that any test makes use of two attributes, so seeing that they sorted correctly (smallest encoded value first) during writing the `SET OF` is possibly missing.

Reply:
I missed the fact that `implicitTag` makes it work with `SequenceOf` without throwing an exception. Since RFC 5208 only specifies BER the sorting doesn't really come into play either. And even if it did then all the current usages do the normalization anyway. So I guess it's not something that really needs additional tests.